source/tests/test-xmlparser.cc

```diff
@@ -74,8 +74,14 @@ int main(int /*argc*/, char *argv[]) {
   int age = parser.constantAs<int>("age", 0);
   assert_test(age == 32);
 
+  std::string incVar1 = parser.constantAs<std::string>("incVar1", "");
+  assert_test(incVar1 == "toto");
+  
+  std::string incVar2 = parser.constantAs<std::string>("incVar2", "");
+  assert_test(incVar2 == "tata");
+
   std::string badguy = parser.constantAs<std::string>("badguy", "");
-  assert_test(badguy == "dracula");
+  assert_test(badguy.empty());
```

Comment:
How was the test passing before? =)

Reply:
Nested constants were processed and this constant was added the list of constants before. Now the constant doesn't exists anymore.

Reply:
Makes sense